lesson02/assignment/css/_variables.scss

```diff
@@ -0,0 +1,5 @@
+//setting few variables for colors
+$light-steel-blue:#B0C4DE;
+$snow:#FFFAFA;
+$ghost-white:#F8F8FF;
+$white-smoke:#F5F5F5;
```

Comment:
Why did these rules need to be in a new variables file, rather than the existing one?